ci/tests/general/general_disk.py

```diff
@@ -114,11 +114,13 @@ def get_physical_disks(client):
             disk_id = disk[0]
             if len(disk_id) > 2 and disk_id[0:2] in ['fd', 'sr', 'lo']:
                 continue
-            if disk[1] in 'disk':
+            types = ['disk', 'raid6']
+            disk[0] = disk[0].decode("ascii", errors="ignore")
```

Comment:
Some comment line containing an example would be nice here why index 3 should '0', why index 1 needs to be 'disk' or 'raid6'